src/org/opendatakit/briefcase/pull/aggregate/AggregateCursor.java

```diff
@@ -128,6 +113,20 @@ public static AggregateCursor of(LocalDate date, String lastReturnedValue) {
     return of(date.atStartOfDay().atOffset(ZoneOffset.UTC), lastReturnedValue);
   }
 
+  private static String buildCursorXml(OffsetDateTime lastUpdate, Optional<String> lastUri) {
+    return String.format("<cursor xmlns=\"http://www.opendatakit.org/cursor\">" +
+            "<attributeName>_LAST_UPDATE_DATE</attributeName>" +
+            "<attributeValue>%s</attributeValue>" +
+            "%s" +
+            "<isForwardCursor>true</isForwardCursor>" +
+            "</cursor>",
+        lastUpdate.format(ISO_OFFSET_DATE_TIME),
+        lastUri
+            .map(value -> "<uriLastReturnedValue>" + value + "</uriLastReturnedValue>")
+            .orElse("<uriLastReturnedValue/>")
```

Comment:
Comment that the `<uriLastReturnedValue/>` must be in that form, to help future readers?